### PR TITLE
Falconsai text summarisation

### PR DESCRIPTION
**Pull request by plutopulp**
A new example for text summarisation.

### CODE REVIEW

File: examples/text-to-text/falconsai-text-summarisation/new_pipeline.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
+import typing as t
+
+from transformers import pipeline as t_pipeline
```
Comment (hirovi) [inline examples/text-to-text/falconsai-text-summarisation/new_pipeline.py:3]: `hf_pipeline` might be more appropiate

Comment (hirovi) [inline examples/text-to-text/falconsai-text-summarisation/new_pipeline.py:3]:
```suggestion
from transformers import pipeline as hf_pipeline
```

File: examples/text-to-text/falconsai-text-summarisation/new_pipeline.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    def resolve_text_to_summarise(self, inputs: Inputs) -> str:
+        if not inputs.file_in and not inputs.text_in:
+            raise Exception("Must either input raw text or a file")
+        if inputs.file_in is not None:
+            return self._read_text(inputs.file_in)
+        return inputs.text_in
+
+    @pipe(run_once=True, on_startup=True)
+    def load(self) -> None:
+        self.summarizer = t_pipeline("summarization", model=HF_MODEL_NAME)
```
Comment (hirovi) [inline examples/text-to-text/falconsai-text-summarisation/new_pipeline.py:52]:
```suggestion
        self.summarizer = hf_pipeline("summarization", model=HF_MODEL_NAME)
```

**Review by hirovi — APPROVED** [review]

File: examples/text-to-text/falconsai-text-summarisation/new_pipeline.py (hunk 1 above)
Comment (plutopulp) [reply examples/text-to-text/falconsai-text-summarisation/new_pipeline.py]: Cool thanks for that, have commited your suggestions

**Pull request merged**